### ISSUE TITLE
MRNet Dataset

### ISSUE THREAD

**Issue by Kyrie-Zhao** [tensorflow/datasets#2386]
* Name of dataset: MRNet Dataset
* URL of dataset: https://stanfordmlgroup.github.io/competitions/mrnet/
* License of dataset: Public Domain
* Short description of dataset and use case(s): The MRNet dataset consisted of 1,370 knee MRI exams and was performed at Stanford University Medical Center between January 1, 2001, and December 31, 2012 (mean age 38.0 years; 569 [41.5%] female patients). The majority vote of 3 musculoskeletal radiologists established reference standard labels on an internal validation set of 120 exams. The dataset contains 1,104 (80.6%) abnormal exams, with 319 (23.3%) ACL tears and 508 (37.1%) meniscal tears; labels were obtained through manual extraction from clinical reports.

I will be working on adding this dataset myself.
